src/System.Runtime.WindowsRuntime/pkg/System.Runtime.WindowsRuntime.pkgproj

```diff
@@ -1,5 +1,8 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <HarvestVersion>4.6.0-preview8.19405.3</HarvestVersion>
```

Comment:
As discussed, we can't harvest preview packages because we need a path to build it.